src/docs/product/integrations/integration-platform/index.mdx

```diff
@@ -8,6 +8,14 @@ description: "Learn more about Sentry’s integration platform and how it provid
 
 Sentry’s integration platform provides a way for external services to interact with Sentry using [webhooks](/product/integrations/integration-platform/webhooks/), [UI components](/product/integrations/integration-platform/ui-components/), and the [REST API](/api). Integrations using this platform are first-class actors within Sentry.
 
+## Quick Start
```

Comment:
This is a misnomer, because "Quick Start" generally means "quick start guide" and those are usually meant to get you up and running in a real world situation, not with a demonstration of one. I'm not married to the title I've suggested, but "quick start" is misleading here. 

Less important, I'm also wondering about whether we should be using the word "demo" vs "reference" when referring to this implementation. Seems like people will immediately know what we mean by "demo" but I'm less sure of "reference."? Probably either is fine, but just wanted to raise that.

```suggestion
## Reference Implementation
```